.github/workflows/ci.yml

```diff
@@ -72,27 +67,70 @@ jobs:
         uses: actions/checkout@v1
       - name: Install system packages
         run: |
-          apt-get update
-          apt-get install -y --no-install-recommends \
-            unzip
+          apk add \
+            unzip \
       - name: Install mongodb PHP extension
         if: matrix.php != '7.1'
         run: |
+          apk add $PHPIZE_DEPS
           pecl install mongodb-1.5.5
           docker-php-ext-enable mongodb
       - name: Disable PHP memory limit
         run: echo 'memory_limit=-1' >> /usr/local/etc/php/php.ini
       - name: Install Composer
-        run: curl -sS -o - https://raw.githubusercontent.com/composer/getcomposer.org/ba13e3fc70f1c66250d1ea7ea4911d593aa1dba5/web/installer | php -- --install-dir=/usr/local/bin --filename=composer --quiet
+        run: wget -qO - https://raw.githubusercontent.com/composer/getcomposer.org/fb22b78362d31c0d2bf516d1f8cdfd2745caa431/web/installer | php -- --install-dir=/usr/local/bin --filename=composer --quiet
       - name: Remove Doctrine MongoDB ODM
         if: matrix.php == '7.1'
         run: |
           composer remove --dev --no-progress --no-update --ansi \
             doctrine/mongodb-odm \
-            doctrine/mongodb-odm-bundle
+            doctrine/mongodb-odm-bundle \
       - name: Update project dependencies
         run: composer update --no-progress --no-suggest --prefer-stable --prefer-lowest --ansi
       - name: Clear test app cache
         run: tests/Fixtures/app/console cache:clear --ansi
       - name: Run PHPUnit tests
         run: vendor/bin/phpunit --colors=always
+
+  phpunit-no-deprecations:
+    name: PHPUnit (PHP ${{ matrix.php }}) (no deprecations)
+    runs-on: ubuntu-latest
+    container:
+      image: php:${{ matrix.php }}-alpine
```

Comment:
any reason to use a docker image here rather than php installed on  `ubuntu-latest`?

Reply:
ah, you are using `7.4-rc` elsewhere which isn't available. ok

Reply:
Because we want full control over the PHP version being used. :smile:

Reply:
ok, but how does using `php:7.3-alpine` give you full control? you need to fix the the patch release for pull control?

Reply:
> how does using php:7.3-alpine give you more control over php7.3 on ubuntu-latest?

For example, there's no `php7.4-rc`. Anyway, it's good to be CI provider-agnostic as much as possible.

> you need to fix the the patch release for full control?

We will do that when necessary. Using the preinstalled version in the VM prevents that.

Reply:
👍